trapper/pipelines/question_answering_pipeline.py

```diff
@@ -374,7 +344,7 @@ def _construct_answer(
 
     def _get_answer_start_ind(self, context, start_token_ind):
         context_tokenized = self.tokenizer(context)["input_ids"]
-        if start_token_ind > len(context_tokenized):
+        if start_token_ind >= len(context_tokenized):
```

Comment:
I think `>` is sufficient, but both comparisons are ok.

Reply:
> that started exactly at the end of the context,

isn't this still a valid output ?

Reply:
Poor choice of words on my part. If the context is N tokens, starting at index N (which would be the token just following the context) shouldn't be valid. But it would pass a strictly greater than N check.